api/docker-compose.develop.yml

```diff
@@ -14,8 +14,10 @@ services:
       - POSTGRES_DB=postgres
       - POSTGRES_PORT=54321
       - POSTGRES_PASSWORD=!ChangeMe!
-      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
-      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+      - KOBO_PASSWORD=${KOBO_PASSWORD:?"Add a password to access KOBO data."}
+      - KOBO_USERNAME=${KOBO_USERNAME:?"Add a user to access KOBO data."}
```

Comment:
If they are declared here you don't need to redeclare them in .deploy if you are not changing the error handling

Reply:
Is that true?  `.deploy` doesn't pick up on `.develop` in the Makefile.

Reply:
Sorry I wasn't clear :)
If most of the files use the same behavior, you could define it in the base docker-compose.yml directly and only override it when needed (eg. for tests)